src/api/app/controllers/staging_project_controller.rb

```diff
@@ -4,7 +4,7 @@ class StagingProjectController < ApplicationController
   private
 
   def set_project
-    @project = Project.find_by(name: params[:project])
+    @project = StagingProject.find_by(name: params[:project])
```

Comment:
I just realized now, while we are here, let's use `find_by!`.

Reply:
:see_no_evil: